newrelic/data_source_newrelic_synthetics_monitor.go

```diff
@@ -0,0 +1,70 @@
+package newrelic
+
+import (
+	"fmt"
+	"log"
+
+	synthetics "github.com/dollarshaveclub/new-relic-synthetics-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceNewRelicSyntheticsMonitor() *schema.Resource {
```

Comment:
Please add a docs page + link to sidebar for the new data source type (see: https://github.com/terraform-providers/terraform-provider-newrelic/blob/master/website/docs/d/application.html.markdown for an example)